pkgs/top-level/python-packages.nix

```diff
@@ -24454,6 +24406,15 @@ EOF
   parse-type = callPackage ../development/python-modules/parse-type { };
 
   ephem = callPackage ../development/python-modules/ephem { };
+
+  jsondate = callPackage ../development/python-modules/jsondate { };
+
+  inflection = callPackage ../development/python-modules/inflection { };
+
+  quandl = callPackage ../development/python-modules/quandl { };
+  # alias for an older package which did not support Python 3
+  Quandl = callPackage ../development/python-modules/quandl { };
```

Comment:
can you keep ordering in this file?

Reply:
We do this to keep merge conflicts low.

Reply:
Done. Not all packages in `python-packages.nix` appear in alphabetical order so I thought it's not important.